python/BUILD

```diff
@@ -49,6 +52,31 @@ py_native_lib_rename(
     visibility = ["//visibility:public"]
 )
 
+# Documentation generation
+py_binary(
+    name = "sphinx_runner",
+    srcs = [
+        "tool/sphinx/sphinx_html_builder_runner.py",
+    ],
+    main = "tool/sphinx/sphinx_html_builder_runner.py",
+    deps = [docs_requirement("sphinx")],
+    visibility = ["//visibility:public"]
+)
```

Comment:
I think we can apply the same idea as in `nodejs` here :) Move this into a BUILD in `tool/sphinx`